README.md

```diff
@@ -26,4 +26,4 @@ fcs = FlowstarContinuousSolution(model)
 See examples/parsing.jl for additional usage.
 
 ## Modeling
-Instead of specifying a model file directly, you can create one programatically as shown in examples/modeling.jl
\ No newline at end of file
+Instead of specifying a model file directly, you can create one programatically as shown in examples/modeling.jl
```

Comment:
```suggestion
Instead of specifying a model file directly, you can create one programatically as shown in [examples/modeling.jl](examples/modeling.jl).
```